packages/dappmanager/src/modules/https-portal/migration.ts

```diff
@@ -18,18 +18,6 @@ const dncoreNetworkName = params.DNP_PRIVATE_NETWORK_NAME;
  * to all running containers.
  */
 export async function addAliasToRunningContainersMigration(): Promise<void> {
-  // Before running all, check DAPPMANAGER
-  const dappmanagerAlias = getPrivateNetworkAlias({
-    dnpName: params.dappmanagerDnpName,
-    serviceName: params.dappmanagerDnpName
-  });
-  const dappmanagerEndpointConfig = await getEndpointConfig(
-    params.dappmanagerContainerName
-  );
-  if (hasAlias(dappmanagerEndpointConfig, dappmanagerAlias)) {
-    return logs.info(`Alias migration already done`);
-  }
```

Comment:
Why remove this? We would run `docker inspect` on all running containers every time the dappmanager starts

Reply:
because there would be an unordered update of docker composes of core packages. I guess the roadmap is to finally have all composes updated with the version 3.5, the alias, and the new network configuration.

In this PR I take the oportunity to also update the dappmanager compose so the migration cannot deppend on the dappmanager. It cannot deppend on any core package since its compose will be randomly updated and we cannot be sure

Reply:
Then what is expected to happen when you run `addAliasToRunningContainersMigration` after a successful migration?

Reply:
discussed